runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -255,34 +255,35 @@ class WorkItemListPage {
   /* Page elements - work item list */
 
   get allWorkItems () {
-    return element.all(by.css(".work-item-list-entry"));
+    return element.all(by.css(".wi-detail-title>p"));
   }
 
   /* xpath = //alm-work-item-list-entry[.//text()[contains(.,'Some Title 6')]]   */
   workItemByTitle (titleString) {
     // return element(by.xpath("//alm-work-item-list-entry[.//text()[contains(.,'" + titleString + "')]]"));
-    return element(by.xpath("//tree-node[.//text()='" + titleString + "']"));
+    return element(by.xpath("//*[@class='wi-detail-title']/p[text()=' " + titleString + " ']"));
   }
 
   get firstWorkItem () {
-    return element.all(by.css(".work-item-list-entry")).first();
+    return element.all(by.css(".wi-detail-title>p")).first();
   }
 
   get lastWorkItem () {
-    return element.all(by.css(".work-item-list-entry")).last();
+    return element.all(by.css(".wi-detail-title>p")).last();
   }
 
   /* Title element relative to a workitem */
-  workItemTitle (workItemElement) {
-    return workItemElement.element(by.css(".f8-wi__list-title")).element(by.css("p")).getText();
+  workItemTitle () {
+    return element.all(by.css(".wi-detail-title>p")).first().getText();
   }
 
   clickWorkItemTitle (title) {
-    return this.clickWorkItem(this.workItemByTitle(title));
+     this.workItemByTitle(title).click()
```

Comment:
`clickWorkItemTitle` should call `clickWorkItem`.

`clickWorkItem` is the generic function. if we wish to click work item by index, then `clickWorkItemByIndex` will search the workitem by the given index, and then call `clickworkItem` with the `workitem` as an argument.